examples/python/README.md

```diff
@@ -5,8 +5,9 @@
 Follow these steps to get started:
 
 1. Get a CDP API key and wallet secret from the [CDP Portal](https://portal.cdp.coinbase.com/access/api)
-2. Fill in your API key and wallet secret in `.env.example`, then run `mv .env.example .env`
-3. Run `source .env` to load the environment variables
+1. Fill in your API key and wallet secret in `.env.example`, then run `mv .env.example .env`
+1. Install dependencies: `uv sync`
+1. Run `source .env` to load the environment variables. You will need to do this every time you open a new terminal
```

Comment:
Micronit: missing period at end

Reply:
Updated for consistency – I saw somewhere once an opinion that bullet points should not end in periods, and I've adopted it ever since 😄